client/pages/home.jsx

```diff
@@ -1,9 +1,27 @@
 import React from 'react';
+import AppContext from '../lib/app-context';
 
-export default function Groups(props) {
-  return (
-    <div>
-      <h1>Home Page</h1>
-    </div>
-  );
+export default class Home extends React.Component {
+
+  render() {
+    const { user } = this.context;
+    // const { firstName } = user;
+    // console.log(user[firstName]);
```

Comment:
Please look through all of your files for commented out code and remove it.

https://github.com/adrianm-arias/final-project/pull/18/files#diff-cc75338894ba5830599852998e0ddb8b3bda389aa0772c5ef479c56842833d14R8-R9

Reply:
Done.